src/music21/base.ts

```diff
@@ -484,6 +484,20 @@ export class Music21Object extends prebase.ProtoM21Object {
             }
         }
     }
+
+    repeatAppend(object, item, numberOfTimes) {
+        let unused = null;
+        try {
+            unused = item.isStream;
+            console.log(item.isStream);
+        } catch (AttributeError) {
+            console.log('to put a non Music21Object in a stream, '
+            + 'create a music21.ElementWrapper for the item');
+        }
+        for (let i = 0; i < numberOfTimes; i++) {
+            object.append(item);
```

Comment:
The `copy.deepcopy(item)` part from music21p is crucial.  In music21j instead you'd write `item.clone(true)` where the `true` means "deep copy" instead of "shallow copy".

(If deep vs. shallow copying is all new -- here's a good article on it :  https://www.geeksforgeeks.org/copy-python-deep-copy-shallow-copy/ )